README.md

```diff
@@ -56,6 +58,43 @@ The report is generated in the run directory and it contains the information abo
 #### Cerberus to help with cluster health checks
 [Cerberus](https://github.com/openshift-scale/cerberus) can be used to monitor the cluster under test and the aggregated go/no-go signal generated by it can be consumed by Kraken to determine pass/fail. This is to make sure the Kubernetes/OpenShift environments are healthy on a cluster level instead of just the targeted components level. It is highly recommended to turn on the Cerberus health check feature avaliable in Kraken after installing and setting up Cerberus. To do that, set cerberus_enabled to True and cerberus_url to the url where Cerberus publishes go/no-go signal in the config file.
 
+### Kubernetes/OpenShift node chaos scenarios supported
+Following node chaos scenarios are supported:
```

Comment:
We might want to mention that the cloud related scenarios like start/stop/reboot are supported only for AWS as of now.

Reply:
Added it.